hermes_eea/Global_Attrs_Cnts.py

```diff
@@ -0,0 +1,133 @@
+import os
+from datetime import datetime
+import math
+
+class Global_Attrs:
+
```

Comment:
What is the purpose of this class? This is missing doc strings.

Reply:
removed